proto/lorenzo/btcstaking/v1/tx.proto

```diff
@@ -63,6 +67,23 @@ message MsgBurnRequest {
 
 message MsgBurnResponse{}
 
+message MsgAddReceiver {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  string authority = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+
+  Receiver receiver = 2 [ (gogoproto.nullable) = false ];
+}
+
+message MsgAddReceiverResponse {}
+message MsgRemoveReceiver {
```

Comment:
`MsgAddReceiver ` and  `MsgRemoveReceiver` should be registered.example:

```go
func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
	// Register messages
	registry.RegisterImplementations(
		(*sdk.Msg)(nil),
		&MsgCreateBTCStaking{},
		&MsgBurnRequest{},
                &MsgAddReceiver{},
                &MsgRemoveReceiver{},
	)

	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
}
```

Reply:
done.